src/CommandHandlers/LoginHandler.cs

```diff
@@ -49,18 +49,6 @@ public override Task Handle(Client client, NetworkObject receivedObject)
         r2.Add(WorldEvent.Get().EventInfoArray(true));
         rl.Add(r2);
 
-        NetworkArray r3 = new();
-        r3.Add(2);
-        r3.Add("LIMBO");
-        r3.Add("default");
-        r3.Add(false);
-        r3.Add(false);
-        r3.Add(false);
-        r3.Add((short)31);
-        r3.Add((short)10000);
-        r3.Add(new NetworkArray());
-        rl.Add(r3);
-
```

Comment:
What does the game do with this data? Is it safe to remove?

Reply:
This is part of room list and looks like is (effectively) ignored by game client. We don't send real room list here, but I think is better do not send LIMBO with invalid room id here.

Probably other entries also can be removed, but I don't remove it, due to the risk of problems in case if this list was empty and set by "ADMIN" room some event variables (which can be set globally and not per room... I don't remember at the moment).